cfgov/unprocessed/css/organisms/chart.less

```diff
@@ -21,7 +21,7 @@
 */
 
 // Import external cfpb-chart-builder stylesheet
-@import (less) 'cfpb-chart-builder.css';
+// @import (less) 'cfpb-chart-builder.css';
```

Comment:
naw, I think the issue isn't this reference. The issue is that  `shrinkwrap.json` file never got updated when package.json changed.  If I run  `npm install` the `cfpb-chart-builder` doesn't get installed ( it does on Travis however ).  @contolini, can you verify this? If so, I'll regenerate the `shrinkwrap.json` file in this PR.